pkg/proxy/client/client_conn.go

```diff
@@ -40,24 +40,25 @@ type ClientConnection struct {
 	nsmgr             *namespace.NamespaceManager
 	ns                *namespace.Namespace
 	connMgr           *backend.BackendConnManager
+	proxyProtocol     bool
 }
-
-func NewClientConnection(logger *zap.Logger, conn net.Conn, frontendTLSConfig *tls.Config, backendTLSConfig *tls.Config, nsmgr *namespace.NamespaceManager, bemgr *backend.BackendConnManager) *ClientConnection {
-	pkt := pnet.NewPacketIOWrapErr(conn, ErrClientConn)
+func NewClientConnection(logger *zap.Logger, conn net.Conn, frontendTLSConfig *tls.Config, backendTLSConfig *tls.Config, nsmgr *namespace.NamespaceManager, bemgr *backend.BackendConnManager, proxyProtocol bool) *ClientConnection {
+	opts := make([]pnet.PacketIOption, 0, 1)
+	if proxyProtocol {
+		opts = append(opts, pnet.WithProxy, pnet.WithClient)
```

Comment:
Never used?

Reply:
I guess something is wrong when merging master...